orchagent/portsorch.cpp

```diff
@@ -4066,6 +4066,15 @@ void PortsOrch::doPortTask(Consumer &consumer)
                             SWSS_LOG_NOTICE("Set port %s SI settings is successful", p.m_alias.c_str());
                             p.m_preemphasis = serdes_attr;
                             m_portList[p.m_alias] = p;
+
```

Comment:
@mihirpat1 there are more than one place where setPortSerdesAttribute() is called. So, its better to notify inside setPortSerdesAttribute()

Reply:
As discussed, since the other place where setPortSerdesAttribute() is called (through initGearboxPort) is triggered only during boot-up and is not directly related to the NPU SI settings being set from xcvrd.
Hence, we will keep the current code as is.